app/models/event.rb

```diff
@@ -20,4 +21,12 @@ class Event < ApplicationRecord
   def service_description
     service_category.service_category_name
   end
+
+  # method to populate estimated_ distance virtual attribute
+  # called from serializer and controller rspec
+  def estimated_distance(loc_lat, loc_long)
```

Comment:
Do we need this method here? Can we just call `Geo.dist_btn_coords(loc_lat, loc_long, pt_latitude, pt_longitude)` directly in the serializer?

Reply:
I had it in the serializer.  In your previous comments you asked me to move it to a model.  I was convinced because as you noted I had to reproduce the logic in the agency controller rspec which was ugly.  Now I can call the method in the model instead of recreating it.

Reply:
Addressed in latest commit